packages/mobile/src/analytics/Properties.tsx

```diff
@@ -577,6 +578,12 @@ interface CeloExchangeEventsProperties {
   }
 }
 
+interface FiatExchangeEventsProperties {
+  [FiatExchangeEvents.external_exchange_link]: {
+    link: string
```

Comment:
Can we pass the exchange name instead of the link? Think that would be easier for us to do analysis on

Reply:
We were going back and forth on what link to include (whether to link directly to the exchange site, or to CELO or cUSD within the exchange site), so I think this detail would be helpful as we might change the link for a given exchange. Can include the exchange too